pagerduty/config.go

```diff
@@ -72,7 +76,12 @@ func (c *Config) Client() (*pagerduty.Client, error) {
 	var httpClient *http.Client
 	httpClient = http.DefaultClient
 	httpClient.Timeout = 1 * time.Minute
-	httpClient.Transport = logging.NewTransport("PagerDuty", http.DefaultTransport)
+
```

Comment:
@dazoakley please add this same update to `Config.SlackClient` 🙏🏽

Reply:
Hi @imjaroiswebdev, thanks for the review! No probs at all updated and rebased. :+1:

Reply:
Thank you!